### DISCUSSION TITLE
not able to train a model to reach accurate values of weight and bias

### DISCUSSION THREAD

**Discussion by AfreenInnovates** [mrdbourke/pytorch-deep-learning#1274] [Q&A]
can anyone help me in understanding why im not able to reach the correct weight and bias values
https://colab.research.google.com/drive/1SVPxBTD4nrRoIT9999JQ9PauerDaBB18?usp=sharing

original values of bias and weight are 0.9 and 0.3 respectively, but after training
OrderedDict([('weight', tensor([0.2417])), ('bias', tensor([0.1388]))])
and if im training for more epochs, its bouncing more, and zig zag curve of loss
